docs/src/componentDocs/Hero/examples/HeroBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {HeroBannerExample} from './HeroBannerExample';
+
+const codeSnippet = `<HeroBanner>
+    <Hero
+        icon={<GradeA size={50}" />}
```

Comment:
There is a error in the code snippet
<img width="507" alt="Screenshot 2024-06-12 at 7 34 53 PM" src="https://github.com/etn-ccis/blui-react-native/assets/130662346/e768aa80-911a-4f72-8549-a07d41f690cc">